app/src/matrix_transform.c

```diff
@@ -13,14 +13,15 @@
 
 #define _TRANSFORM_ENTRY(i, _)                                                                     \
     [(KT_ROW(DT_PROP_BY_IDX(ZMK_KEYMAP_TRANSFORM_NODE, map, i)) * ZMK_MATRIX_COLS) +               \
-        KT_COL(DT_PROP_BY_IDX(ZMK_KEYMAP_TRANSFORM_NODE, map, i))] = i,
+        KT_COL(DT_PROP_BY_IDX(ZMK_KEYMAP_TRANSFORM_NODE, map, i))] = i + 1,
```

Comment:
Consider making a `#define` for the `1` here. That would ensure that the same offset is added here as is subtracted later, and it could help make the code easier to understand if given a descriptive name.

Reply:
DONE